src/main/java/domain/BlackjackParticipant.java

```diff
@@ -0,0 +1,83 @@
+package domain;
+
+import except.BlackJackException;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class BlackjackParticipant {
+
+    private static final int BURST_STANDARD = 21;
+    private static final int ACE_DIFF = 10;
+    private static final String INVALID_CARD_STATE = "비정상적인 카드 추가입니다. 플레이어는 21장 이상 받을 수 없습니다";
+    private static final String INVALID_NAME = "닉네임은 공백일 수 없습니다";
+    private static final String DEALER_NAME = "딜러";
+    private static final String INVALID_PLAYER_NAME = "일반 플레이어는 딜러일 수 없습니다.";
+
+    private final List<TrumpCard> trumpCards = new ArrayList<>();
+    private final String name;
+
+    protected BlackjackParticipant(String name) {
+        this.name = name;
+        validateNickname();
+    }
+
+    protected static String dealerName() {
+        return DEALER_NAME;
+    }
+
+    private void validateNickname() {
+        if (name == null || name.isBlank()) {
+            throw new BlackJackException(INVALID_NAME);
+        }
+    }
+
+    public void addDraw(TrumpCard trumpCard) {
+        if (isBurst(calculateCardSum())) {
+            throw new IllegalStateException(INVALID_CARD_STATE);
+        }
+        trumpCards.add(trumpCard);
+    }
+
+    public int calculateCardSum() {
+        int sum = trumpCards.stream().map(TrumpCard::cardNumberValue)
```

Comment:
IntStream 을 활용해보죠 :)

```suggestion
        int sum = trumpCards.stream().mapToInt(TrumpCard::cardNumberValue)
```

Reply:
아 intStream을 map으로 사용하겠다고 명시할수있었군요 꿀팁 감사합니다!